docs/mobility/slam/cartographer/_index.md

```diff
@@ -13,21 +13,24 @@ aliases:
 # SMEs: Kat, Jeremy
 ---
 
+{{% alert title="Warning" color="warning" %}}
+
+Running `cartographer` in the cloud is temporarily disabled, and the attribute `use_cloud_slam` defaults to `false`.
+If you set `use_cloud_slam` to `true`, it will not be possible to see the results in the remote control card.
+
+{{% /alert %}}
+
 [The Cartographer Project](https://github.com/cartographer-project) contains a C++ library that performs dense Simultaneous Localization And Mapping (SLAM).
 
 To use Cartographer with the Viam {{< glossary_tooltip term_id="slam" text="SLAM" >}} service, you can use the [`cartographer`](https://app.viam.com/module/viam/cartographer) {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}}.
 See the [Use Modules](/registry/#use-modules) section for instructions on using a module from the Viam registry on your machine.
 
-This module utilizes the Cartographer SLAM algorithm to provide tools for generating SLAM maps.
-If you want to use a SLAM algorithm outside of Cartographer, you can create a new module to suit your needs.
```

Comment:
I found the first sentence here redundant with the sentence on L16 above, and second sentence confusing - at first read I thought it was suggesting making another module that wraps Cartographer. Just suggesting removing these two sentences.

Reply:
I think we want to say here that you could use a different algorithm and would need to create that as a modular resource. You could do that, right?

Reply:
Yes, definitely, I just think it's not the clearest as written and didn't think it's very important to mention this here since it's part of how Viam's platform works